Classes/Utility/Curl/Request.php

```diff
@@ -79,6 +79,22 @@ public function get()
         return $this->executeRequest($request);
     }
 
+    /**
+     * delete data from a defined URL
+     *
+     * @return Response
+     */
+    public function delete()
```

Comment:
Please annotate the return value here

Reply:
Done, and went over the whole file to correct return types and the header comment